server/service/src/static_files.rs

```diff
@@ -65,19 +138,19 @@ impl StaticFileService {
 }
 
 /// Returns the file name part of the path like:
-/// `./static_file_path/{ui}_{file_name};
-fn parse_original_file_name(id: &str, file_path: &PathBuf) -> Option<String> {
+/// `./static_file_path/{uuid}_{file_name};
+fn parse_original_file_name(id: &str, file_path: &Path) -> Option<String> {
     let file_name = file_path.file_name()?.to_string_lossy();
     let name = &file_name[id.len() + 1..];
-    if name.len() == 0 {
+    if name.is_empty() {
```

Comment:
Some of these are changes from HSH code based on clippy suggestions.
Bit of copying over happening here...